content/en/docs/setup/production-environment/windows/user-guide-windows-nodes.md

```diff
@@ -180,9 +197,28 @@ All code snippets in Windows sections are to be run in a PowerShell environment
     [Environment]::SetEnvironmentVariable("HTTPS_PROXY", "http://proxy.example.com:443/", [EnvironmentVariableTarget]::Machine)
     ```
 
-    If after reboot you see the following error, you need to restart the docker service manually
+    After reboot, you can check via below command to check your docker service is ready.
```

Comment:
Revise the sentence please, there are two "check" in it, which reads strange.

Reply:
Thanks. I'll apply.